.nvmrc

```
@@ -0,0 +1 @@
+v16.13.0
```

no need for a precise version
```suggestion
16
```

---

I just tested with your proposition, and it works 👍🏻